modules/ROOT/pages/_partials/flex-tls-cipher.adoc

```diff
@@ -0,0 +1,69 @@
+//tag::connectedSelect[]
+
+== Select Ciphers
+
+When you configure a TLS context, Secrets Manager applies default ciphers based on the TLS versions you select. In addition to the defaults, you can select other ciphers to use with the selected TLS version. Each TLS context can have multiple ciphers.
```

Comment:
"In addition to the defaults": Note that it is possible for the user to remove default ciphers as well as to add new ones